### PR TITLE
Enable arm image build

### PR DESCRIPTION
**Pull request by kanya-approve**

### CODE REVIEW

**Comment by kanya-approve** [comment]
@westsurname

File: .github/workflows/docker-build-push.yml [hunk 1]
```diff
@@ -35,7 +35,10 @@ jobs:
         id: meta
         uses: docker/metadata-action@v5
         with:
-          images: ${{ matrix.image }}         
+          images: ${{ matrix.image }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
```
Comment (westsurname) [inline .github/workflows/docker-build-push.yml:41]: Can you explain this?

**Comment by westsurname** [comment]
@kanya-approve I made a change to the action in attempt to get it to run for this pull request, can you merge that change in and push again?

File: .github/workflows/docker-build-push.yml (hunk 1 above)
Comment (kanya-approve) [reply .github/workflows/docker-build-push.yml:41]: https://github.com/marketplace/actions/build-and-push-docker-images

It's to allow building against more platforms

**Comment by kanya-approve** [comment]
> @kanya-approve I made a change to the action in attempt to get it to run for this pull request, can you merge that change in and push again?

done

**Comment by westsurname** [comment]
Merging as is because I can't get the images to build. Will roll back if broken.

**Pull request merged**